Dockerfile

```diff
@@ -1,10 +1,12 @@
 # Build image with DPDK, etc.
-FROM debian:12-slim AS builder
+FROM --platform=${TARGETPLATFORM} debian:12-slim AS builder
```

Comment:
This prints a warning for the default `docker build .` command. Not sure if this can be solved easily. The build itself seems to work though.

Reply:
Could you please use the docker ` buildx` to build ? This is the recommended way to build multi architecture docker images anyways. The command would be then `docker buildx build .`
Does buildx work fine ? buildx should automatically provide the builtin variables like "TARGETPLATFROM" or TARGETARCH or it can be also specified with "--platform linux/amd64" in local builds.

Reply:
Cannot directly test on docker as we are using `podman` everywhere. In podman at least `buildx` is just an alias and the output is the same. But as image compilation work with just this warning (which in CI/CD will be solved), it's not a blocker.

Reply:
I would see this point as "acceptable" then as it doesnt break the podman workflow and we use buildx with github workflows so the CI/CD in github actions needs to cope with docker and buildx.